src/SFA.DAS.AdminService.Web.UnitTests/Controllers/Home/HomeControllerTests.cs

```diff
@@ -0,0 +1,101 @@
+﻿using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.AdminService.Settings;
+using SFA.DAS.AdminService.Web.Controllers;
+using SFA.DAS.AdminService.Web.Models;
+using System.Security.Claims;
+using FluentAssertions.Execution;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.AdminService.Web.UnitTests.Controllers.Home
+{
+    [TestFixture]
+    public class HomeControllerTests
+    {
+        private HomeController _controller;
+        private Mock<IWebConfiguration> _mockWebConfiguration;
+
+        [SetUp]
+        public void Setup()
+        {
+            _mockWebConfiguration = new Mock<IWebConfiguration>();
+        }
+
+
+        [TestCase(true)]
+        [TestCase(false)]
+        public void Then_Index_Returns_ViewModel(bool useDfESignIn)
+        {
+            //arrange
+            _mockWebConfiguration.Setup(c => c.UseDfESignIn).Returns(useDfESignIn);
+            _controller = new HomeController(_mockWebConfiguration.Object)
+            {
+                ControllerContext = new ControllerContext { HttpContext = new DefaultHttpContext() },
+            };
+
+            //sut
+            var result = _controller.Index();
+
+            //assert
+            result.Should().NotBeNull();
+            var resultModel = result.Should().BeOfType<ViewResult>().
+                Which.Model.Should().BeOfType<HomeViewModel>().Which;
+            resultModel.UseDfESignIn.Should().Be(useDfESignIn);
```

Comment:
These asserts should be wrapped in a `using (new AssertionScope())` so the test does not abort on first failed assertion and the test results are accurately reported